eternalcore-core/src/main/java/com/eternalcode/core/configuration/implementation/PluginConfiguration.java

```diff
@@ -92,6 +92,9 @@ public static class Teleport implements SpawnSettings {
         public Duration teleportationTimeToSpawn() {
             return this.teleportTimeToSpawn;
         }
+
```

Comment:
mógłbyś dodać jeszcze checka czy ma sprawdzać WorldBorder jeżeli wb jest ustawiony to pobiera radius borderowy w innym przypadku radius z configu